packages/api-client/src/team/search/TeamSearchOptions.ts

```diff
@@ -29,7 +29,7 @@ export interface TeamSearchOptions {
   /** Filter results by member role */
   frole?: Role[];
   /** Sort order (asc | desc | undefined) */
-  order?: SearchOrder;
+  sortorder?: SearchOrder;
   /** Max number of search results. Defaults to 15 results. Min 1, max 500. */
```

Comment:
I guess these kind of comments instead of `//` is for the IDEs to show descriptions?

Reply:
yes